events_task/test/my_app/accounts/accounts_test.exs

```diff
@@ -6,11 +6,11 @@ defmodule MyApp.AccountsTest do
   describe "users" do
     alias MyApp.Accounts.User
 
-    @valid_attrs %{email: "some email"}
-    @update_attrs %{email: "some updated email"}
+    @valid_attrs %{email: "sergei@gmail.com"}
```

Comment:
It's better to test with 'fake' data when possible - by 'fake' I mean data that won't belong to any real person.
This is to protect us from situation when somebody accidentally launched test code in production env - we don't want to actually send an email to `sergei@gmail.com` :).
IANA reserved `example.com` domain for this purpose - it's better to use it in tests.